reservations/models.py

```diff
@@ -340,6 +351,9 @@ class Reservation(models.Model):
         verbose_name=_("Details for this reservation's cancellation"), blank=True
     )
 
+    created_at = models.DateTimeField(
+        verbose_name=_("Created at"), null=True, default=timezone.now
```

Comment:
`timezone.now` is a function object => it does not call it but rather returns the function. So changing it to be like `default=timezone.now()` would do the trick but I like to use django's `auto_now_add` so instead of giving default value we say: `auto_now_add=True` would be nice.

But that said, I'm thinking that does it use the default timezone :thinking:

Reply:
oh, checking this bit further and it seems that the `default=timezone.now` should work
So django must know it is a callable and calls when necessary.

Reply:
I'll try to explain my reasoning for choosing `default=timezone.now` here:

I originally used `auto_now_add`, but the problem with it is that the field is not as easily testable. For example, you can't do `ReservationFactory(created_at=...)` because whatever you pass to the factory will always be overwritten by the `auto_now_add` with the current time. Then you have to start mocking the dates with `freezegun` which is not so neat.

With `default=timezone.now` we can just pass whatever time we want to the factory and testing becomes a lot easier. 🙂 So I went for the option that facilitates more convenient testing.